operator/pkg/helmreconciler/render.go

```diff
@@ -345,3 +386,11 @@ func toChartManifestsMap(m name.ManifestMap) ChartManifestsMap {
 	}
 	return out
 }
+
+// progressBar returns a progress bar for a number of items.
+func progressBar(items int) *pb.ProgressBar {
+	bar := pb.StartNew(items)
+	bar.SetTemplateString(`{{counters . }} {{bar . }} {{percent . }} {{etime . "%s"}}`)
```

Comment:
I think we can improve this format a bit, but I like the idea. Lets try out some ideas in a followup and get some feedback